README.md

```diff
@@ -5,8 +5,7 @@ This Firmware Wizard lets a user select the correct firmware for his device. Dir
 
 Similar projects:
 - [Freifunk Bielefeld Firmware Wizard](https://github.com/freifunk-bielefeld/firmware-wizard/): Based on this wizard, but also supports LEDE and OpenWRT firmware images
-- [Freifunk Hennef Firmware Downloader](https://github.com/Freifunk-Hennef/ffhef-fw-dl): Firmware wizard that also contains server-side code (PHP)
-- [LibreMesh Chef](https://chef.libremesh.org/): Firmware wizard of LibreMesh that supports building custom images on demand
+- [LibreMesh Chef](https://github.com/libremesh/chef): Firmware wizard of LibreMesh that supports building custom images on demand
```

Comment:
not sure about LibreMesh Chef either:

"This repository has been archived by the owner on Mar 15, 2021. It is now read-only."

```
WARNING: this project is no longer mainted. Please checkout it's successors:

https://gitlab.com/openwrt/web/firmware-selector-openwrt-org
https://github.com/sudhanshu16/openwrt-firmware-selector
```

Reply:
I've got no strong opionion on it. Let's just go with your proposal.